### ISSUE TITLE
mkwindows.sh fails on armv7l using Debian due to hardcoded ar binary name in tgl

### ISSUE THREAD

**Issue by v4Lo** [majn/telegram-purple#454]
The tgl `Makefile.in` has `ar` binary name hardcoded in line 66. This prevents cross-compilation for Windows using the `mkwindows.sh` script on armv7l using Debian in latest dev-1.4.0 branch.
```
tgl/libs/libtgl.a: error adding symbols: Archive has no index; run ranlib to add one
collect2: error: ld returned 1 exit status
Makefile:148: recipe for target 'bin/libtelegram.dll' failed
```

Changing `ar` to `i686-w64-mingw32-ar` (or a template) resolves this issue and successfully enables cross-compilation on the above mentioned platform.

I don't know whether this should be fixed here or in tgl.

**Comment by BenWiederhake** [comment]
Yup, you're right!

Also, damn github! Only close it when the change has landed.

**Comment by BenWiederhake** [comment]
You should be able to try it out now.

**Comment by v4Lo** [comment]
It's working as intended now. Thank you.

**Comment by BenWiederhake** [comment]
Should be fixed in [v1.4.1](https://github.com/majn/telegram-purple/releases/tag/v1.4.1).
